### PR TITLE
remove "wrap" verbiage for using Stellar assets in smart contracts

### PR DESCRIPTION
**Pull request by briwylde08**
Fixes: https://github.com/stellar/stellar-docs/issues/301

### CODE REVIEW

File: docs/fundamentals-and-concepts/stellar-data-structures/assets.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ Accounts on the Stellar network can be used to track, hold, and transfer any typ
 
 Assets on Stellar have two identifying characteristics: the asset code and the issuer. Since more than one organization can issue a credit representing the same asset, asset codes often overlap (for example, multiple companies offer a USD token on Stellar). Assets are uniquely identified by the combination of their asset code and issuer.
 
-You can also create smart contract tokens using the [Token Interface](https://soroban.stellar.org/docs/reference/interfaces/token-interface) but it is possible, and even recommended in most cases, to [wrap a Stellar asset](#wrapping-stellar-assets) using the Stellar Asset Contract for use in smart contracts.
+You can also create smart contract tokens using the [Token Interface](https://soroban.stellar.org/docs/reference/interfaces/token-interface) but it is possible, and even recommended in most cases, to implement the [Stellar Asset Contract](#using-stellar-assets-in-smart-contracts) to interact with Stellar assets in smart contracts.
```
Comment (leighmcculloch) [inline docs/fundamentals-and-concepts/stellar-data-structures/assets.mdx:12]: I think this paragraph belongs in its own section at the very bottom of this page, a section titled "Custom Token Contracts".

And saying something like:
> Custom token contracts can be deployed on Stellar by deploying a contract that implements the [Token Interface](https://soroban.stellar.org/docs/reference/interfaces/token-interface), which is the same interface implemented by the Stellar Asset Contract for Stellar Assets.

File: docs/fundamentals-and-concepts/stellar-data-structures/assets.mdx [hunk 2]
```diff
@@ -27,9 +27,9 @@ There is no dedicated operation to create an asset on Stellar. Instead, assets a
 
 The public key of the issuing account is linked on the ledger to the asset. Responsibility for and control over an asset resides with the issuing account. Since settings are stored at the account level on the ledger, the issuing account is where you use set_options operations to link to meta-information about an asset and set authorization flags.
 
-## Wrapping Stellar assets
+## Using Stellar assets in smart contracts
 
-Assets issued on the Stellar network can be used in smart contracts with the Stellar Asset Contract (SAC), which is an implementation of [CAP-46-6: Smart Contract Standardized Asset](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0046-06.md). The SAC allows users to use their Stellar account and trustline balances in Soroban and is a primary intersection where smart contracts interact with the rest of the Stellar network.
+The Soroban CLI can deploy a Stellar Asset Contract (SAC) for a Stellar asset that allows any Soroban smart contract to interact with that asset. Deploying the SAC for a Stellar asset enables that asset for use with smart contracts.
```
Comment (leighmcculloch) [inline docs/fundamentals-and-concepts/stellar-data-structures/assets.mdx:32]: I think this section could do with the lead in explanation:

```suggestion
Assets issued on the Stellar network are accessible to smart contracts. Every Stellar asset has reserved a Stellar Asset Contract that can be deployed by anyone who wants to be able to interact with the asset from a contract.

The Soroban CLI can deploy a Stellar Asset Contract for a Stellar asset. Deploying the Stellar Asset Contract for a Stellar asset enables that asset for use with smart contracts.
```

**Review by leighmcculloch — COMMENTED** [review]
Couple suggestions. Wdyt?

**Comment by briwylde08** [comment]
Thanks, @leighmcculloch! Your suggestions have been implemented.

**Review by ElliotFriend — APPROVED** [review]
Looks good to me!

**Pull request merged**